Polyhedron/demo/Polyhedron/Use_ssh.cpp

```diff
@@ -0,0 +1,247 @@
+#ifdef CGAL_USE_SSH
```

Comment:
license?

Reply:
@maxGimeno Most of that file is probably a copy-paste from somewhere. We should check on which license we can put this file.

Reply:
Do you remember the source of the copy-paste?

Reply:
It is all base don the guide provided by libssh

Reply:
The put the file under the license `LGPLv3+`.